coap-gateway/service/refreshToken.go

```diff
@@ -38,11 +39,10 @@ func validateRefreshToken(req CoapRefreshTokenReq) error {
 	return nil
 }
 
-func validUntilToExpiresIn(v int64) int64 {
-	if v == 0 {
+func validUntilToExpiresIn(validUntil time.Time) int64 {
```

Comment:
refresh token request could be handled directly by the new package authorization (now named provider). what do you think?